django-stubs/core/checks/registry.pyi

```diff
@@ -19,7 +19,10 @@ class Tags:
     translation: str
     urls: str
 
-_CheckCallable: TypeAlias = Callable[..., Sequence[CheckMessage]]
+class _CheckCallable(Protocol):
+    def __call__(
+        self, *, app_configs: Apps, databases: Sequence[str] | None, **kwargs: Any
```

Comment:
`app_configs` has the wrong type - it's a list of `AppConfig`, or `None`.

See https://docs.djangoproject.com/en/4.1/topics/checks/#writing-your-own-checks

> The check function must accept an app_configs argument; this argument is the list of applications that should be inspected. If None, the check must be run on all installed apps in the project.

And an example check function in one of my packages: https://github.com/adamchainz/django-linear-migrations/blob/main/src/django_linear_migrations/apps.py#L129

Reply:
Doh that was sloppy. Thanks for the fixes!